addon/components/form-field.js

```diff
@@ -67,7 +67,7 @@ export default Component.extend(formFieldOptions, {
       let propertyPath = this.get('propertyPath');
 
       if (!this.get('disabled') && typeof this.attrs.updated === 'function') {
-        this.attrs.updated(object, propertyPath, value);
+        this.attrs.updated(object, propertyPath, value, ...arguments);
```

Comment:
`arguments` are all the arguments passed to this update function, so including the value.
If you did write a test for this, you knew it would fai

```js
function(value, ...args) {
  updated(object, propertyPath, value, ...args);
}
```

It will only work when you don't need the value and pass everything by using spread.
```js
function() {
  updated(object, propertyPath, ...arguments);
}
```